src/ui/qml/NetworkSetupWizard.qml

```diff
@@ -93,43 +93,110 @@ ApplicationWindow {
         id: firstPage
 
         Column {
-            spacing: 8
+            width: parent.width
+            spacing: 20
+            anchors {
+                left: parent.left
+                right: parent.right
+                margins: parent.width/10
+            }
+
 
-            Label {
+            Column {
                 width: parent.width
-                text: qsTr("This computer's Internet connection is free of obstacles. I would like to connect directly to the Tor network.")
-                wrapMode: Text.Wrap
-                horizontalAlignment: Qt.AlignHCenter
-            }
+                spacing: 10
 
-            Button {
-                anchors.horizontalCenter: parent.horizontalCenter
-                text: qsTr("Connect")
-                isDefault: true
-                onClicked: {
-                    // Reset to defaults and proceed to bootstrap page
-                    configPage.reset()
-                    configPage.save()
+                Image {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    source: "../icons/ricochet.svg"
+                    sourceSize.width: parent.width / 10
```

Comment:
It's not a good idea to use bindings for sourceSize. Every time it changes, the image will be reloaded (so every time parent.width changes..) which can cause a bad UX on resize (bad performance)